public/robots.txt

```diff
@@ -1,5 +1,11 @@
 User-agent: *
-Disallow: /users/password/edit
-Disallow: /sign_up/enter_password
-Disallow: /sign_up/email/confirm
-Disallow: /api/saml/
+Disallow: /
+Allow: /$
+Allow: /es$
+Allow: /fr$
```

Comment:
Do we want to try to do this programmatically with https://github.com/18F/identity-idp/blob/4087aba3d76b0607fdf7f441781ff581d203ba0b/lib/idp/constants.rb#L3?

Reply:
Yeah, you're probably right, we could do this with a custom route + controller and it'd probably be better, and it would also let us use URL route helpers as well.

Reply:
> are you suggesting making that a live file served by a Rails controller, or an ERB that we write out to `public/` as a build step?

I was thinking a controller, and implemented a first pass in 3ee1ae8, though I do like the idea of a static file since it wouldn't be something we expect to change. Then again, we probably don't get much traffic to this so hopefully it's not a big deal either way?